flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mongodb-cdc/src/test/java/org/apache/flink/cdc/connectors/mongodb/source/NewlyAddedTableITCase.java

```diff
@@ -798,7 +798,7 @@ private void testNewlyAddedCollectionOneByOne(
             fetchedDataList.addAll(expectedSnapshotDataThisRound);
             waitForUpsertSinkSize("sink", fetchedDataList.size());
             MongoDBAssertUtils.assertEqualsInAnyOrder(
-                    fetchedDataList, TestValuesTableFactory.getResults("sink"));
+                    fetchedDataList, TestValuesTableFactory.getRawResultsAsStrings("sink"));
```

Comment:
`getResults` => `getResultsAsStrings`?

This issue also presents in many other places